### PR TITLE
July 26th Upstream Merge

### PR DESCRIPTION
**Pull request by Jolly-66**
Somethings broken and I gotta check it

### CODE REVIEW

**Comment by Jolly-66** [comment]
game refuses to power itself. funny

**Comment by Jolly-66** [comment]
Disabled the BO beret, and commented out its paths in its respective files. Will need to be converted, sometime.

File: tgstation.dme [hunk 1]
```diff
@@ -855,7 +862,6 @@
 #include "code\datums\status_effects\status_effect.dm"
 #include "code\datums\status_effects\wound_effects.dm"
 #include "code\datums\weather\weather.dm"
-#include "code\datums\weather\weather_types\acid_rain.dm"
```
Comment (MrMelbert) [inline tgstation.dme:858]: this file is unticked now and can be deleted

File: tgstation.dme [hunk 2]
```diff
@@ -631,7 +637,6 @@
 #include "code\datums\diseases\_disease.dm"
 #include "code\datums\diseases\_MobProcs.dm"
 #include "code\datums\diseases\anxiety.dm"
-#include "code\datums\diseases\appendicitis.dm"
```
Comment (MrMelbert) [inline tgstation.dme:634]: this file is unticked now and can be deleted

File: tgstation.dme [hunk 3]
```diff
@@ -2161,7 +2180,6 @@
 #include "code\modules\events\space_ninja.dm"
 #include "code\modules\events\spacevine.dm"
 #include "code\modules\events\spider_infestation.dm"
-#include "code\modules\events\spontaneous_appendicitis.dm"
```
Comment (MrMelbert) [inline tgstation.dme:2164]: this file is unticked now and can be deleted

File: tgstation.dme [hunk 4]
```diff
@@ -2310,13 +2330,12 @@
 #include "code\modules\hydroponics\grown\eggplant.dm"
 #include "code\modules\hydroponics\grown\flowers.dm"
 #include "code\modules\hydroponics\grown\garlic.dm"
+#include "code\modules\hydroponics\grown\gatfruit.dm"
 #include "code\modules\hydroponics\grown\grass_carpet.dm"
+#include "code\modules\hydroponics\grown\hedges.dm"
 #include "code\modules\hydroponics\grown\korta_nut.dm"
-#include "code\modules\hydroponics\grown\kudzu.dm"
```
Comment (MrMelbert) [inline tgstation.dme:2315]: this file is unticked now and can be deleted

File: tgstation.dme [hunk 5]
```diff
@@ -2310,13 +2330,12 @@
...
 #include "code\modules\hydroponics\grown\melon.dm"
-#include "code\modules\hydroponics\grown\misc.dm"
 #include "code\modules\hydroponics\grown\mushrooms.dm"
-#include "code\modules\hydroponics\grown\nettle.dm"
```
Comment (MrMelbert) [inline tgstation.dme:2319]: this file is unticked now and can be deleted

File: tgstation.dme [hunk 6]
```diff
@@ -1944,8 +1958,12 @@
 #include "code\modules\buildmode\submodes\smite.dm"
 #include "code\modules\buildmode\submodes\throwing.dm"
 #include "code\modules\buildmode\submodes\variable_edit.dm"
-#include "code\modules\capture_the_flag\capture_the_flag.dm"
-#include "code\modules\capture_the_flag\map_loading.dm"
```
Comment (MrMelbert) [inline tgstation.dme:1948]: these files are unticked now and can be deleted

File: code/modules/admin/verbs/mapping.dm [hunk 7]
```diff
@@ -149,17 +136,16 @@ GLOBAL_LIST_EMPTY(dirty_vars)
...
 	if(intercom_range_display_status)
-		for(var/obj/item/radio/intercom/I in world)
-			for(var/turf/T in orange(7,I))
-				var/obj/effect/debugging/marker/F = new/obj/effect/debugging/marker(T)
-				if (!(F in view(7,I.loc)))
-					qdel(F)
+		for(var/frequency in GLOB.all_radios)
+			for(var/obj/item/radio/intercom/intercom in GLOB.all_radios[frequency])
+				for(var/turf/turf in view(7,intercom.loc))
+					var/atom/marker = new /obj/effect/abstract/marker/intercom(turf)
```
Comment (MrMelbert) [inline code/modules/admin/verbs/mapping.dm:148]:
```suggestion
					new /obj/effect/abstract/marker/intercom(turf)
```
this is safe to remove for our sanity and it's going to happen upstream as well

File: jollystation_modules/code/modules/jobs/job_types/bridge_officer.dm [hunk 8]
```diff
@@ -57,7 +57,7 @@
 	gloves = /obj/item/clothing/gloves/color/black
 	uniform = /obj/item/clothing/under/rank/security/bridge_officer/black
 	shoes = /obj/item/clothing/shoes/laceup
-	head = /obj/item/clothing/head/beret/black/bridge_officer
+	// head = /obj/item/clothing/head/beret/black/bridge_officer - Upstream Merge 26; Beret needs to be converted to GAGs
```
Comment (MrMelbert) [inline jollystation_modules/code/modules/jobs/job_types/bridge_officer.dm:60]: the BO sprites should still work despite the beret changes, just add
```
	greyscale_config = null
	greyscale_config_worn = null
```
to the definition of the BO beret and it'll work like old until we convert it properly

**Review by MrMelbert — COMMENTED** [review]
Something we should've been more keen on doing which is becoming more noticeable as of now is removing files that were deleted, since git doesn't do that manually on a merge. At some point i'll automate it but for now, while not necessary to do this it maintains a bit of order

File: jollystation_modules/code/modules/jobs/job_types/bridge_officer.dm (hunk 8 above)
Comment (Jolly-66) [reply jollystation_modules/code/modules/jobs/job_types/bridge_officer.dm:60]: I tested in game and it ERROR'd on me

File: tgstation.dme (hunk 1 above)
Comment (Jolly-66) [reply tgstation.dme:858]: keeping this for the future

**Pull request merged**